client.py

```diff
@@ -34,6 +23,67 @@
         if iteration_counter == 10:
             iteration_counter = 0
             print("Map size:", my_map.size())
+
+
+def sql_example(hz_client):
+    print("Creating a mapping...")
+    # See: https://docs.hazelcast.com/hazelcast/5.0/sql/mapping-to-maps
+    mapping_query = "CREATE MAPPING cities TYPE IMap OPTIONS ('keyFormat'='varchar','valueFormat'='varchar')"
+    hz_client.sql.execute(mapping_query)
+    print("The mapping has been created successfully.")
+    print("--------------------")
+
+    print("Inserting data via SQL...")
+    insert_query = """
+    INSERT INTO cities VALUES
+    ('Australia','Canberra'),
+    ('Croatia','Zagreb'),
+    ('Czech Republic','Prague'),
+    ('England','London'),
+    ('Turkey','Ankara'),
+    ('United States','Washington, DC');
+    """
+    hz_client.sql.execute(insert_query)
+    print("The data has been inserted successfully.")
+    print("--------------------")
+
+    print("Retrieving all the data via SQL...")
+    result = hz_client.sql.execute("SELECT * FROM cities").result()
+    for row in result:
+        country = row.get_object_with_index(0)
+        city = row.get_object_with_index(1)
```

Comment:
```suggestion
        country = row[0]
        city = row[1]
```
You can also use that easier way to access columns

Reply:
Fixed